.murdock

```diff
@@ -16,7 +16,6 @@ tests/driver_h* tests/driver_i* tests/driver_j* tests/driver_my9221"}
 
 export RIOT_CI_BUILD=1
 export CC_NOCOLOR=1
-export STATIC_TESTS=${STATIC_TESTS:-1}
```

Comment:
Maybe go the soft integration step first and just default this to `0` and remove the code at some later point?

Reply:
Done